src/mozilla_taskgraph/transforms/scriptworker/shipit/mark_as_shipped.py

```diff
@@ -0,0 +1,37 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import os
+
+from taskgraph.transforms.base import TransformSequence
+
+transforms = TransformSequence()
+
+
+@transforms.add
+def make_task_description(config, tasks):
+    if "shipit" not in config.graph_config:
+        config_path = os.path.join(config.graph_config.root_dir, "config.yml")
+        raise Exception(f"Missing 'shipit' config in {config_path}")
+
+    shipit = config.graph_config["shipit"]
+    release_format = shipit.get(
+        "release-format", "{product}-{version}-build{build_number}"
```

Comment:
I noticed that the release name in Ship-It ended up being `App-services-...` and that the API is case sensitive. Are all releases capitalized? If so should I ensure `product` is capitalized here?

Reply:
(alternatively we can set the product to `App-services` manually in config.yml, but might be nice to avoid that footgun)

Reply:
If I remember correctly, releases are always capitalized. I don't recall whether this is a big deal or not. I.e.: shipit rejects it for being case sensitive.

Reply:
It does. Well the API endpoint returns a 404. Maybe this could be fixed on the shipit side (make the endpoint case insensitive).

In the meantime, sounds like capitalizing here could help us avoid hitting this issue in the future.

Reply:
Nvm, capitalizing doesn't play nicely with custom release formats so I'm just going to leave it as is for now.